mkdumprd

```diff
@@ -319,10 +319,41 @@ check_crypt()
 	done
 }
 
+temp_dracut_confdir=
+# This is a workaround for dracut argument inheritance not doing
+# what one would expect. xref https://issues.redhat.com/browse/RHEL-49590
+# and https://github.com/rhkdump/kdump-utils/issues/11
+maybe_synthesize_tmp_confdir()
+{
+	local confs
+	confs=$(shopt -s nullglob; echo /usr/lib/dracut/dracut.conf.d/*.conf)
+	if test -z "${confs}"; then
+		return
+	fi
+	temp_dracut_confdir=$(mktemp -d -t kdump.XXXXXX)
+	for c in ${confs}; do
+		if grep -qEe '^dracutmodules.*=' "${c}"; then
```

Comment:
Hi, it would be better to skip both dracutmodules and omit_dracutmodules as we have the other issue when omit_dracutmodules includes nfs then there's no way to add it for the time being.